pricing.ftd

```diff
@@ -6,6 +6,7 @@
 
 
 -- lib.page:
+active-page: pricing/
```

Comment:
@priyanka9634 as mentioned above comment - plz try to avoid additional properties to `page` component. Try a different approach.

Reply:
this will fixed after footer sitemap